src/main/kotlin/uk/gov/justice/digital/hmpps/hmppstier/config/ObjectMapperConfiguration.kt

```diff
@@ -1,27 +0,0 @@
-package uk.gov.justice.digital.hmpps.hmppstier.config
-
-import com.fasterxml.jackson.annotation.JsonInclude
-import com.fasterxml.jackson.databind.DeserializationFeature
-import com.fasterxml.jackson.databind.ObjectMapper
-import com.fasterxml.jackson.databind.SerializationFeature
-import com.fasterxml.jackson.datatype.jdk8.Jdk8Module
-import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule
-import com.fasterxml.jackson.module.kotlin.KotlinModule
-import org.springframework.context.annotation.Bean
-import org.springframework.context.annotation.Configuration
-import org.springframework.context.annotation.Primary
-
-@Configuration
-class ObjectMapperConfiguration {
-
-  @Bean(name = ["globalObjectMapper"])
-  @Primary
-  fun objectMapper(): ObjectMapper {
```

Comment:
Has this been eplaced or is it not needed any more?!

Reply:
Spring initialises this through config https://github.com/ministryofjustice/hmpps-tier/blob/main/src/main/resources/application.yml#L7 so this wasn't needed

Reply:
👍